packages/client/hmi-client/src/workflow/operator/tera-operator-header.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	<nav :class="status">
+		<h5>{{ name }}</h5>
+		<Button
+			icon="pi pi-ellipsis-v"
+			class="p-button-icon-only p-button-text p-button-rounded"
+			@click="toggleMenu"
+		/>
+		<Menu ref="menu" :model="options" :popup="true" />
+	</nav>
+</template>
+
+<script setup lang="ts">
+import { ref, PropType } from 'vue';
+import { OperatorStatusType } from '@/types/operator';
+import Button from 'primevue/button';
+import Menu from 'primevue/menu';
+
+const emit = defineEmits(['remove-node', 'open-in-new-window']);
+
+defineProps({
+	name: {
+		type: String,
+		default: ''
+	},
+	status: {
+		type: Object as PropType<OperatorStatusType>,
+		default: OperatorStatusType.DEFAULT
+	}
+});
+
+const menu = ref();
+const toggleMenu = (event) => {
+	menu.value.toggle(event);
+};
+
+function bringToFront() {
+	// TODO: bring to front
+	// maybe there can be a z-index variable in the parent component
+	// and we can just increment it here, and add a z-index style to the node
+	// console.log('bring to front');
+}
+
+const options = ref([
+	{ icon: 'pi pi-clone', label: 'Duplicate', command: () => bringToFront },
+	{
+		icon: 'pi pi-external-link',
+		label: 'Open in new window',
+		command: () => emit('open-in-new-window')
+	},
+	{ icon: 'pi pi-arrow-up', label: 'Bring to front', command: () => bringToFront },
+	{ icon: 'pi pi-arrow-down', label: 'Bring to back', command: () => bringToFront },
+	{ icon: 'pi pi-trash', label: 'Remove', command: () => emit('remove-node') }
+]);
+</script>
+
+<style scoped>
+nav {
+	display: flex;
+	padding: 0.5rem;
+	height: 32px;
```

Comment:
1rem in this project is 14px not 16px so that's not going to work. One thing I suspect is that in the figma 1rem = 16px but that's not the same here.

Reply:
So we have to be careful here, Figma is set to `16px` per `1rem` because that's how we set it up last time with @jamiewaese-uncharted and I when opening Dev mode for the first time on your laptop.
I'm keen to update `1rem = 16px` on the whole app to be consistent with Figma.